pycm/pycm_util.py

```diff
@@ -211,6 +242,37 @@ def normalized_table_calc(classes, table):
     return new_table
 
 
+def sparse_matrix_calc(classes, table):
```

Comment:
Duplication :

Duplicate of `sparse_table_calc` function.
`table` and `matrix`are one.